eng/pipelines/templates/steps/typespec-ci.yml

```diff
@@ -14,7 +14,14 @@ steps:
     pushd ${{parameters.Folder}}
 
     # Pass "--no" to ensure npx does not install anything (TypeSpec compiler should already be installed)
-    npx --no tsp compile . --warn-as-error || exit_code=1
+    if test -f tspconfig.yaml; then
```

Comment:
```suggestion
```

You can remove this check.  We should only call `templates/steps/typespec-ci.yml` on folders already known to contain `tspconfig.yaml`.